pkg/operator/ceph/object/rgw.go

```diff
@@ -101,6 +89,14 @@ func (c *clusterConfig) startRGWPods() error {
 		c.store.Spec.Gateway.Instances = 1
 	}
 
+	// Create the controller owner ref
+	// It will be associated to all resources of the CephObjectStore
+	ref, err := opcontroller.GetControllerObjectOwnerReference(c.store, c.scheme)
```

Comment:
Seems odd that an err would be returned from creating the owner reference. We already have the CR with the `c.store`. What could go wrong? Is there a lookup for some other object?

Reply:
Yes, GVKForObject()